src/test/perf_local.cc

```diff
@@ -612,10 +637,20 @@ static inline void prefetch(const void *object, uint64_t num_bytes)
 }
 #endif
 
+#if defined(__aarch64__) // || defined(__arm__)
```

Comment:
nit, since this implementation is mutual exclusive with the `HAVE_SSE` one, could you use `#elif` instead?